template.yaml

```diff
@@ -417,6 +417,31 @@ Resources:
       Tags:
         - Key: Name
           Value: OrchSessionTable
+      ResourcePolicy:
+        PolicyDocument:
+          Version: 2012-10-17
+          Statement:
+            - Sid: AllowOrchsessionTableAccessForLogout
+              Effect: Allow
+              Action:
+                - dynamodb:DescribeTable
+                - dynamodb:DeleteItem
+              Resource: "*"
```

Comment:
Should this be `Resource:  !GetAtt OrchSessionTable.Arn`?

Reply:
Because it's a resource policy and is attached to the orch session table, that gives a circular dependency error. `Resource: "*"` (I _believe_) just means every resource, which in this case only means the dynamoDb table.

It seems I can have no `Resource` property. I'll push that change and see if that works in testing

Reply:
[Does a `DependsOn` attribute help fix the circular dependency? ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-dependson.html)

Reply:
Wait that wont work, as is, but maybe if the resource policy isn't defined inline

Reply:
Yeah it looks a little strange but this is the correct way of writing it